mkdocs_enumerate_headings_plugin/plugin.py

```diff
@@ -175,6 +178,9 @@ def on_post_page(self, output, page, config, **kwargs):
             )
             return output
 
+        if self.config.get("enumerate_nav", True):
+            output = output.replace(f"<h1>{page.chapter}. ", f"<h1>")
```

Comment:
Shouldn't this be ` if not` instead? Because you want to only remove it when `enumerate_nav` is false

Reply:
What is removed here is inserted by `enumerate__nav` in the first place -- the "double enumeration" on pages without an h1. So if `enumerate_nav` is disabled, there should be nothing to remove here.